timely_beliefs/visualization/ridgeline.py

```diff
@@ -0,0 +1,233 @@
+    std2 = np.array([(float(pred_temp_2[i])-float(pred_temp_05[i]))/(np.sqrt(2)*erfinv((2*cp_list[1])-1)) for i in range(len(pred_temp_05))])
+    std = (std1+std2)/2
+    #plot everything
+    show_plot(mean, std, start, end, fixedviewpoint)
+
+
+def show_plot(mean, std, start, end, fixedviewpoint=False):
+    """
+    Creates and shows ridgeline plot
+    @param mean: list of float, mean values
+    @param std: list of float, standard deviation values
+    @param start: int,start hours before event-time
+    @param end: int,end hours before event-time
+    @param fixedviewpoint : BOOLEAN, if true create fixed viewpoint plot
+    """
+    #amount of lines to draw
+    nr_lines = len(mean)
+    x = np.linspace(-10, 30, 500)
+    frame = pd.DataFrame()
+    #generate points for each line
+    for i in range(nr_lines):
+        frame["{}".format(i)] = stats.norm.pdf(x, mean[i], std[i])
+    #set color pallete to viridis
+    pallete = viridis(nr_lines)
+    #set list reversed or not depending on viewpoint
+    if fixedviewpoint:
+        cats = list(frame.keys())
+    else:
+        cats = list(reversed(frame.keys()))
+    source = ColumnDataSource(data=dict(x=x))
+    #creat figure
+    p = figure(y_range=cats, plot_width=900, x_range=(-5, 30), toolbar_location=None)
+    #make a ridge line in the figure
+    for i, cat in enumerate(reversed(cats)):
+        y = ridge(cat, frame[cat], 50)
+        source.add(y, cat)
+        p.patch('x', cat, alpha=0.6, color=pallete[i], line_color="black", source=source)
+    #added y axis with the right ticks etc depending on fixedviewpoint
+    if fixedviewpoint:
+        p.yaxis.axis_label = 'Number of hours after event_start'
+        y_ticks = list(np.arange(0, nr_lines, 5))
+        yaxis = LinearAxis(ticker=y_ticks)
+        y_labels = list((np.arange(0, nr_lines, 5)))
+    else:
+        p.yaxis.axis_label = 'Number of hours before event_start'
+        y_ticks = list(np.arange(end, 0, -5))
+        yaxis = LinearAxis(ticker=y_ticks)
+        y_labels = list(np.arange(start, end, 5))
+    #map ticks on to a dict
+    mapping_dict = {y_ticks[i]: str(y_labels[i]) for i in range(len(y_labels))}
+    for i in range(nr_lines):
+        if i not in mapping_dict:
+            mapping_dict[i]=" "
+    mapping_code = "var mapping = {};\n    return mapping[tick];\n    ".format(mapping_dict)
+    #format ticks
+    p.yaxis.formatter = FuncTickFormatter(code=mapping_code)
+    #set plot atributes
+    p.outline_line_color = None
+    p.background_fill_color = "#ffffff"
+    p.xaxis.ticker = FixedTicker(ticks=list(range(-20, 101, 10)))
+    p.xaxis.axis_label = 'Temperature (Celcius)'
+    p.ygrid.grid_line_color = None
+    p.xgrid.grid_line_color = "#000000"
+    p.xgrid.ticker = p.xaxis[0].ticker
+    p.axis.minor_tick_line_color = None
+    p.axis.major_tick_line_color = None
+    p.axis.axis_line_color = None
+    #add padding in y derection
+    p.y_range.range_padding = 0.2 / (nr_lines / 168)
+    show(p)
```

Comment:
You could probably set the x-axis range nicely, too. I've used `p.x_range` = Range1d(-5, 25)` for now, but it's of course better to base the interval on the data. Right now the plot shows a lot of empty space on the right hand side.